examples/flutter_firebaseauth/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: flutter_firebaseauth
+description: Flutter Firebase Auth using bloc library
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.1.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  cupertino_icons: ^0.1.2
+  flutter_bloc: any
```

Comment:
can we reference the local `flutter_bloc` just like the other examples:

```yaml
flutter_bloc:
  path: ../../packages/flutter_bloc
```

Reply:
Done.